src/libsyntax/feature_gate.rs

```diff
@@ -228,6 +228,9 @@ declare_features! (
     // Allows comparing raw pointers during const eval
     (active, const_compare_raw_pointers, "1.27.0", Some(53020), None),
 
+    // Allows panicking during const eval (produces compile-time errors)
+    (active, const_panic, "1.29.0", Some(51999), None),
```

Comment:
This number probably needs updating (and I thought things were sorted chronologically, not alphabetically, heh).